avahi/avahi.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/usr/sbin/avahi-daemon --no-drop-root &
```

Comment:
Why is this in a separate file and not in docker-startup.sh?

Reply:
This is going to be changed in next PR anyway. All-in-one.

I had some difficulties with avahi-daemon and for some reason it started better this way, but... there was an issues still.
Next entrypoint script will have `/usr/sbin/avahi-daemon -k`for resatarting avahi-daemon. This will fix issue that I saw with restarts. After this fix both dbus and avahi-daemon are restarted.